Uplift/Views/Onboarding/OnboardingView.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  OnboardingView.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 11/18/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class OnboardingView: UIView {
+
+    // MARK: - Private vars
+    private let imageView = UIImageView()
+    private let titleLabel = UILabel()
+    private var tableView: UITableView?
```

Comment:
Can we make the tableView a `let` variable? If there are no gym names available for whatever reason we can just hide the table view.

Reply:
The onboarding views at the beginning and end never display gyms/classes, which makes them never have a table view, so I made it conditional to communicate that